index.html

```diff
@@ -2,23 +2,28 @@
 
 <head>
     <title></title>
-    <link rel="stylesheet" type="text/css" href="css/reset" />
+    <link rel="stylesheet" type="text/css" href="css/reset.css" />
     <link rel="stylesheet" type="text/css" href="css/base.css" />
     <link rel="stylesheet" type="text/css" href="css/layout.css" />
     <link rel="stylesheet" type="text/css" href="css/modules.css" />
+    <link rel="stylesheet" type="text/css" href="css/theme.css" />
 </head>
 
 <body>
     <main>
-        <section id="a" class="header">A</section>
-        <section id="b" class="sidebar-left-lg">B</section>
-        <section id="c" class="2-col-lg">C</section>
-        <section id="d" class="2-col-lg">D</section>
-        <section id="e" class="2-col 4-col-lg">E</section>
-        <section id="f" class="2-col 4-col-lg">F</section>
-        <section id="g" class="2-col 4-col-lg">G</section>
-        <section id="h" class="2-col 4-col-lg">H</section>
-        <section id="i" class="fullwidth">I</section>
+        <header id="a">A</header>
+        <aside id="b" class="">B</aside>
```

Comment:
Generally, the aside could be inside or outside of the main section. I typically default to it being outside of main unless I have a clear reason why it seems like it is a part of the core/main content of the site.